static/src/javascripts/projects/common/modules/commercial/user-features.js

```diff
@@ -71,6 +73,12 @@ const persistResponse = (JsonResponse: () => void) => {
         HIDE_SUPPORT_MESSAGING_COOKIE,
         !JsonResponse.showSupportMessaging
     );
+    if (JsonResponse.oneOffContributionDate) {
+        const date = Date.parse(JsonResponse.oneOffContributionDate);
+        if (!Number.isNaN(date)) {
+            addCookie(SUPPORT_ONE_OFF_CONTRIBUTION_COOKIE, date.toString());
```

Comment:
Is this a session cookie?

Reply:
after chatting with @twrichards we've decided to make this a separate cookie, with the same lifecycle as the other cookies set using members-data-api